edweiss-app/app/(app)/todo/edit.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file edit.tsx
+ * @description Screen component for editing a todo item in the edweiss app
+ * @author Adamm Alaoui
+ */
+
+// ------------------------------------------------------------
+// --------------- Import Modules & Components ----------------
+// ------------------------------------------------------------
+
+import { ApplicationRoute } from '@/constants/Component';
+import Todolist from '@/model/todo';
+import { useLocalSearchParams } from 'expo-router';
+import { AbstractTodoEditor } from '../../../components/todo/abstractTodoEditor';
+
+// Types
+type Todo = Todolist.Todo;
+
+
+// ------------------------------------------------------------
+// -------------------   Todo Edit Screen   -------------------
+// ------------------------------------------------------------
+
+const EditTodoScreen: ApplicationRoute = () => {
```

Comment:
That a thing I didn't though of. I will look into it for in future iterations so that it feels more responsive and alive from the standpoint of the end user.